tests/template/srw_generate_data/gaussian-x-ray-beam-through-a-beamline-containing-imperfect-mirrors.py

```diff
@@ -389,6 +389,21 @@ def set_optics(v, names=None, want_final_propagation=True):
 
 
 
+def epilogue():
```

Comment:
Epilogues only run in the server, not when user is running manually. I think we want to insert a simple code in the parameters.py only when run on our server:

```py
import sirepo.template
sirepo.template.run_epilogue('srw')
```

This will have access to in.json, for example, which run.py would not.

Reply:
4b8ceee19